DonjinKrawler-Client/src/main/java/donjinkrawler/Player.java

```diff
@@ -37,6 +41,11 @@
     private AudioPlayer audioPlayer = new AudioPlayer();
     private PlayerCommander commander = new PlayerCommander();
     private Boolean backwards = false;
+    private Image attackIMG;
+    private double damage = 20;
+    private Boolean attack = false;
```

Comment:
Instead of using the object type, use the primitive type - boolean (lowercase **b**). Same for every other Boolean out there. No real reason to use the reference type.

Reply:
:(